sonic_api.yaml

```diff
@@ -1400,3 +1676,10 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  NeighborEntry:
+    type: object
+    properties:
+      neighbor_ip:
+        type: string
+      neighbor_as:
+        type: integer
```

Comment:
This needs to be unit.

Reply:
Couldn't find any uint data type for yaml. It only has integers.
https://swagger.io/docs/specification/data-models/data-types/